channels/views/channels_test.py

```diff
@@ -56,10 +55,11 @@ def test_list_channels_anonymous(client, settings, allow_anonymous):
         # Until we decide otherwise this should always be an empty list.
         assert resp.json() == []
     else:
-        assert resp.status_code == status.HTTP_401_UNAUTHORIZED
+        assert resp.status_code in (status.HTTP_401_UNAUTHORIZED, status.HTTP_403_FORBIDDEN)
+        assert resp.data['error_type'] == 'NotAuthenticated'
```

Comment:
Can you declare constants for the HTTP status code tuple, `'NotAuthenticated'`, and `'PermissionDenied'`? At a glance it seems like it should be easy enough to make those changes via find/replace, but if it's not and it would take a while to do, don't worry about it